policy/pa.go

```diff
@@ -141,6 +142,7 @@ var (
 	errLabelTooShort       = berrors.MalformedError("DNS label is too short")
 	errLabelTooLong        = berrors.MalformedError("DNS label is too long")
 	errMalformedIDN        = berrors.MalformedError("DNS label contains malformed punycode")
+	errInvalidRLDH         = berrors.RejectedIdentifierError("DNS name contains is a R-LDH label")
```

Comment:
Typo: "DNS name contains is a R-LDH label".

Should that be "DNS name contains a R-LDH label"  ?